src/components/MoreInformation.js

```diff
@@ -53,12 +55,100 @@ function ExtraData({ data }) {
     } else if (typeof data == "object") {
         let elements = [];
         for (const key in data) {
-            elements.push(
-                <div className={classes.extraData}>
-                    <b>{key}:</b> {data[key]}
-                </div>
-            );
+            //formats additional information instead of just adding it to the more information section
+            if (key === "Additional Information") {
+                const currentData = data[key];
+                if (currentData) {
+                    var finalData = parseMD(currentData);
+                    //adds the parsed data as straight html
+                    elements.push(
+                        <div
+                            className={classes.extraData}
+                            dangerouslySetInnerHTML={{
+                                __html: `<b>${key}:</b> ${finalData}`,
+                            }}
+                        />
+                    );
+                }
+            } else {
+                elements.push(
+                    <div className={classes.extraData}>
+                        <b>{key}:</b> {data[key]}
+                    </div>
+                );
+            }
         }
         return elements;
     }
 }
+
+function parseMD(currentData) {
+    //encodes angle brackets to escape data
+    let workingData1;
+    let workingData2;
+    let indexOfLAB = currentData.indexOf("<");
+    let indexOfRAB = currentData.indexOf(">");
+    if (indexOfLAB !== -1) {
+        workingData1 = currentData.replaceAll("<", "&#60;");
+    } else {
+        workingData1 = currentData;
+    }
+    if (indexOfRAB !== -1) {
+        workingData2 = workingData1.replaceAll(">", "&#62;");
+    } else {
+        workingData2 = workingData1;
+    }
+    //parses bold italics first, splitting data at each "***"
+    let splitDataEmStr = workingData2.split("***");
+    let newDataEmStr = splitDataEmStr[0];
+    let isEmStr = false;
+    //iterates through the split data
+    //uses em to italicize and span with the secondary color for bolding (went with a color change since the titles were bold)
+    let i = 0;
+    for (i = 1; i < splitDataEmStr.length; i++) {
+        if (isEmStr) {
+            newDataEmStr = newDataEmStr + "</span></em>" + splitDataEmStr[i];
+            isEmStr = false;
+        } else {
+            newDataEmStr =
+                newDataEmStr +
+                '<em><span style="color: #F50057">' +
+                splitDataEmStr[i];
+            isEmStr = true;
+        }
```

Comment:
Optional: you could simplify a little by using template strings + wrapping the em in one pass:

```suggestion
        if (isEmStr) {
            newDataEmStr = newDataEmStr + splitDataEmStr[i];
            isEmStr = false;
        } else {
            newDataEmStr = newDataEmStr + `<em><span style="color: #F50057">${splitDataEmStr[i]}</span></em>`;
            isEmStr = true;
        }
```

Reply:
To add to this, using += would make it even more concise while still retaining clarity.